pontoon/base/tests/managers/test_entity.py

```diff
@@ -1080,6 +1080,357 @@ def test_mgr_entity_filter_combined(admin, resource_a, locale_a, user_a):
     )
 
 
+@pytest.mark.django_db
+def test_mgr_entity_option_match_case(admin, resource_a, locale_a, user_a):
+    entities = [
+        EntityFactory.create(
+            key="key %s" % i,
+            resource=resource_a,
+            string="TestEntity%s" % i,
+        )
+        for i in range(0, 2)
+    ]
+
+    TranslationFactory.create(
+        locale=locale_a,
+        entity=entities[0],
+        rejected=True,
+        user=user_a,
+    )
+    TranslationFactory.create(
+        locale=locale_a,
+        entity=entities[1],
+        approved=True,
+        user=user_a,
+    )
+
+    # Base case
+    assert list(
+        Entity.for_project_locale(
+            admin,
+            resource_a.project,
+            locale_a,
+            search="testentity",
+            author=user_a.email,
```

Comment:
Ideally, we should construct Base test as:

```python
assert list(Entity.for_project_locale(*args, **kwargs)) == ...
```

And then we'd just modiffy kwargs accordingly. That would mean less repetition and less chance for mistakes.